src/components/WebexSettings/Select.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
there is no need for a dedicated story for the Select component, we are not inviting developers to use it, it's just for internal use.

Reply:
Ok, I'll remove it.